cmd/deploy/command_configuration_test.go

```diff
@@ -92,7 +92,11 @@ devs:
 		t.Fatal("error trying to get configmap from data object")
 	}
 
-	assert.Equal(t, expectedCfg, currentCfg)
+	assert.Equal(t, expectedCfg.Name, currentCfg.Name)
+	assert.Equal(t, expectedCfg.Namespace, currentCfg.Namespace)
+	assert.Equal(t, expectedCfg.Labels, currentCfg.Labels)
+	assert.Equal(t, expectedCfg.Data, currentCfg.Data)
+	assert.NotNil(t, currentCfg.Annotations[model.LastUpdatedAnnotation])
```

Comment:
If I'm not wrong, `currentCfg.Annotations[model.LastUpdatedAnnotation]` cannot be nil as Annotations is a `map[string]string` and the default value would be an empty string. So this assert would always be true even if the annotation is not present.

There would be 2 ways of checking this:
* `_, ok := currentCfg.Annotations[model.LastUpdatedAnnotation]` and check that `ok` is true
* `value := currentCfg.Annotations[model.LastUpdatedAnnotation]` and verify that `val != ""`

Same about the tests in `deploy_test.go` and `translate_test.go`